packages/mux-video/src/index.ts

```diff
@@ -66,14 +64,12 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElement> implements Pa
     return [...AttributeNameValues, ...(CustomVideoElement.observedAttributes ?? [])];
   }
 
-  // Keeping this named "__hls" since it's exposed for unadvertised "advanced usage" via getter that assumes specifically hls.js (CJP)
-  protected __hls?: PlaybackEngine;
+  protected __core?: PlaybackCore;
   protected __playerInitTime: number;
   protected __metadata: Readonly<Metadata> = {};
   protected __playerSoftwareVersion?: string;
   protected __playerSoftwareName?: string;
-  protected __updateAutoplay?: UpdateAutoplay;
-  protected __errorTranslator?: Function;
+  protected __errorTranslator?: (errorEvent: any) => any;
```

Comment:
nitpick(non-blocking): If we're updating the type to be more precise, can this be stricter than `any` for the param & the yield? Feels like it should.